lib/tasks/ensure-no-unsupported-deps.js

```diff
@@ -11,8 +12,38 @@ export default async function ensureNoUnsupportedDeps() {
       'Your app contains a dependency to ember-fetch. ember-fetch behaves a way that is incompatible with modern JavaScript tooling, including building with Vite.',
     );
     console.log(
-      'Please remove ember-fetch dependency then run this codemod again. Check out https://github.com/emberjs/rfcs/pull/1065 to see recommended alternatives.',
+      'Please remove ember-fetch dependency then run this codemod again. Check out https://github.com/emberjs/rfcs/pull/1065 to see recommended alternatives.\n',
     );
+    shouldExit = true;
+  }
+
+  const hasEmberComposableHelpers =
+    packageJSON['dependencies']?.['ember-composable-helpers'] ||
+    packageJSON['devDependencies']?.['ember-composable-helpers'];
+  if (hasEmberComposableHelpers) {
+    console.log(
+      'Your app contains a dependency to ember-composable-helpers. ember-composable-helpers contains a "won\'t fix" Bablel issue that makes it icompatible with Vite.',
+    );
+    console.log(
+      'Please move from the original ember-composable-helpers to @nullvoxpopuli/ember-composable-helpers then run this codemod again.\n',
+    );
+    shouldExit = true;
+  }
+
+  const hasEmberCliMirage =
+    packageJSON['dependencies']?.['ember-cli-mirages'] ||
+    packageJSON['devDependencies']?.['ember-cli-mirage'];
+  if (hasEmberCliMirage) {
+    console.log(
+      "Your app contains a dependency to ember-cli-mirage. ember-cli-mirage doesn't work correctly with Vite.",
+    );
+    console.log(
+      'Please move from ember-cli-mirage to ember-mirage then run this codemod again.\n',
```

Comment:
we should link to the docs here, https://github.com/bgantzler/ember-mirage/blob/main/docs/migration.md

Even if the docs aren't 100% right they can be improved later 👍